modules/luci-base/htdocs/luci-static/resources/ui.js

```diff
@@ -2265,8 +2266,106 @@ var UIDynamicList = UIElement.extend(/** @lends LuCI.ui.DynamicList.prototype */
 			this.addItem(dl, this.values[i], label);
 		}
 
+		this.initDragAndDrop(dl);
+
 		return this.bind(dl);
 	},
+	initDragAndDrop: function(dl) {
+		let draggedItem = null;
+		let touchStartY = 0;
+		let placeholder = null;
+
+		dl.addEventListener('dragstart', (e) => {
+			if (e.target.classList.contains('item')) {
+				draggedItem = e.target;
+				e.target.style.opacity = 0.5;
+			}
+		});
+
+		dl.addEventListener('dragend', (e) => {
+			if (e.target.classList.contains('item')) {
+				e.target.style.opacity = "";
+			}
+		});
+
+		dl.addEventListener('dragover', (e) => {
+			e.preventDefault();
+		});
+
+		dl.addEventListener('dragenter', (e) => {
+			if (e.target.classList.contains('item')) {
+				e.target.style.borderTop = "2px solid #aaa";
+			}
+		});
+
+		dl.addEventListener('dragleave', (e) => {
+			if (e.target.classList.contains('item')) {
+				e.target.style.borderTop = "";
+			}
+		});
```

Comment:
maybe
```js
dl.addEventListener('dragleave', (e) => resetStyles(e.target));
```

Reply:
See previous comment

Reply:
I removed the "if", but the resetStyles does not (always) reset the opacity.